dbcon/joblist/tuplehashjoin.h

```diff
@@ -620,6 +620,8 @@ class TupleHashJoinStep : public JobStep, public TupleDeliveryStep
   int64_t djsSmallLimit;
   int64_t djsLargeLimit;
   uint64_t djsPartitionSize;
+  uint32_t djsMaxPartitionTreeDepth;
```

Comment:
According with the style guide we add a suffix '_' to an attribute,e.g canSplit_.

Reply:
I agree, but all other variables in this code has different style, so we will have zoo of variables with different styles, how about keep this style, until we change them everywhere.